src/config/database.js

```diff
@@ -2,17 +2,8 @@ import pg from "pg";
 import logger from "../utils/logger.js";
 import { DEV_DATABASE_URL, DATABASE_URL, NODE_ENV } from "./constants.js";
 import dotenv from "dotenv";
-dotenv.config();
-const pool = new pg.Pool({
```

Comment:
It looks like you had a challenge with using a database URL. I can help with that instead of what you've done here - commenting out the existing code to put in what works for you.